packages/salesforcedx-vscode-core/README.md

```diff
@@ -1,4 +1,4 @@
-# salesforcedx-vscode-core
+# Salesforce CLI integration for Visual Studio Code
```

Comment:
I see that "integration" isn't capitalized in the actual extension's name, but it should be. Is it possible to change that at this point, @vazexqi ?

Reply:
Yep, will do. @ntotten , please capitalize this here and I'll update it in the `package.json`.